cms/djangoapps/contentstore/utils.py

```diff
@@ -295,9 +295,10 @@ def get_group_display_name(user_partitions, xblock_display_name):
         group name (String): Group name of the matching group.
     """
     for user_partition in user_partitions:
-        for group in user_partition['groups']:
-            if str(group['id']) in xblock_display_name:
-                return group['name']
+        if user_partition['scheme'] == 'random':
```

Comment:
@cahrens We can't use `get_split_user_partitions` method because it requires `UserPartition` as argument, here we have dict containing user partitions and their info. 

I can get partitions but that would be an extra call because `create_xblock_info` method already calculates and uses `user_partitions` that we are passing to our method `get_group_display_name(user_partitions, xblock_display_name)`